app/src/main/java/com/infomaniak/mail/ui/MainActivity.kt

```diff
@@ -53,14 +53,15 @@ class MainActivity : ThemedActivity() {
 
     private val backgroundColor: Int by lazy { getColor(R.color.backgroundColor) }
     private val backgroundHeaderColor: Int by lazy { getColor(R.color.backgroundHeaderColor) }
+    private val menuDrawerBackground: Int by lazy { getColor(R.color.menuDrawerBackground) }
```

Comment:
Why not naming it menuDrawerBackgroundColor or backgroundMenuDrawerColor to be consistent ?

Reply:
`backgroundMenuDrawerColor` 👌

Reply:
```suggestion
    private val backgroundMenuDrawerColor: Int by lazy { getColor(R.color.backgroundMenuDrawerColor) }
```

Reply:
And `R.color.backgroundMenuDrawerColor` need to be renamed accordingly.